Lib/json/tool.py

```diff
@@ -25,22 +25,22 @@ def main():
                         help='a JSON file to be validated or pretty-printed')
     parser.add_argument('outfile', nargs='?', type=argparse.FileType('w'),
                         help='write the output of infile to outfile')
-    parser.add_argument('--sort-keys', action='store_true', default=False,
-                        help='sort the output of dictionaries alphabetically by key')
+    parser.add_argument('--sort-keys', action='store_true',
+                        help='sort dictionary outputs alphabetically by key')
```

Comment:
@nirs I changed it to:

> sort dictionaries output alphabetically by key

as per your suggestion. But would it make more sense as possessive:

> sort dictionaries' output alphabetically by key